ci/pipelines/deploy-to-production.yml

```diff
@@ -392,6 +403,93 @@ jobs:
       - set_pipeline: deploy-to-production
         file: deploy-to-prod-pipeline-definition/ci/pipelines/deploy-to-production.yml
 
+  - name: deploy-egress-to-prod
+    serial: true
+    serial_groups: [deploy-application]
+    plan:
+      - get: egress-ecr-registry-prod
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: egress-ecr-registry-prod/tag
+      - task: create-notification-snippets
+        file: pay-ci/ci/tasks/create-egress-notification-snippets.yml
+        params:
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          ENV: production-2
+      - load_var: failure_snippet
+        file: snippet/failure
+      - load_var: success_snippet
+        file: snippet/success
+      - load_var: start_snippet
+        file: snippet/start    
+      - <<: *put_start_slack_notification  
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_prod_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-prod-assume-role
+      - load_var: role
+        file: assume-role/assume-role.json
+        format: json
+      - task: check-release-versions
+        file: pay-ci/ci/tasks/check-release-versions.yml
+        params:
+          <<: *aws_assumed_role_creds
+          AWS_REGION: "eu-west-1"
+          CLUSTER_NAME: "production-2-fargate"
+          APP_NAME: egress
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+      - task: deploy-to-prod
+        file: pay-ci/ci/tasks/deploy-egress.yml
+        params:
+          <<: *aws_assumed_role_creds
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          ACCOUNT: production
+          ENVIRONMENT: production-2
+      - task: wait-for-deploy
+        file: pay-ci/ci/tasks/wait-for-deploy.yml
+        params:
+          APP_NAME: egress
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          <<: *aws_assumed_role_creds
+          ENVIRONMENT: production-2
+    <<: *put_success_slack_notification_p1      
+    <<: *put_failure_slack_notification
+
+  - name: smoke-test-egress-on-prod
+    serial_groups: [smoke-test]
+    plan:
+      - get: egress-ecr-registry-prod
+        trigger: true
+        passed: [deploy-egress-to-prod]
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: egress-ecr-registry-prod/tag
+      - task: create-notification-snippets
+        file: pay-ci/ci/tasks/create-notification-snippets.yml
```

Comment:
Just noticed this (missed it on the test/staging pipelines). I think this will need to be `create-egress-notification-snippets.yml` otherwise [the links in Slack will be broken](https://gds.slack.com/archives/CEBPUR2JE/p1629718763023800) (and have the `ACTION_NAME` param passed in so we know it's a smoke test).